### ISSUE TITLE
Normalization of data to custom values

### ISSUE THREAD

**Issue by claradelarosa96** [satijalab/seurat#5855]
Hello,

I have a dataset where half my cells are a control condition and half a disease condition. All the data is already scaled and normalized, quality controls done etc. I was wondering if there is any way in which I could normalize all the cells across features by a control value, which I would provide. Essentially, I would want to be able to see and analyze the data after a certain point in my analysis always already relative to control, without having to carry in parallel control and disease conditions. 

In other words, I would like to do something like per cluster disease cells, divide all features across all disease cells by the average of the feature in the control cells of the cluster, for instance.

I have been looking through Seurat functions and I don't seem to find a way. I've also tried accessing directly object@assay$RNA@scale.data and object@assay$RNA@data to modify them directly but it also doesn't seem to work too well.

I'm fairly new to Seurat, so I thank you in advance for all the possible help :)
